networkit/cpp/Unittests-X.cpp

```diff
@@ -75,13 +75,14 @@ int main(int argc, char **argv) {
 	// feenableexcept(FE_ALL_EXCEPT);
 #endif
 
-	std::string program_name = argv[0];
-	// PARSE OPTIONS
-	argc-=(argc>0); argv+=(argc>0); // skip program name argv[0] if present
+	std::string program_name(argc ? argv[0] : "");
```

Comment:
Is there really any OS where the program name is *not* in argv/argc?

Reply:
I also wondered that. The standard says that *if* `argc>0` then `argv[0]` is the program name. So we cannot assume `argc > 0` (even if I'm not aware of any OS that doesn't pass the name).  But in the end, it seems really not worth to introduce UB to get rid of three simple checks.